runtime/compiler/z/codegen/J9S390PrivateLinkage.cpp

```diff
@@ -2768,7 +2768,7 @@ void TR::J9S390JNILinkage::checkException(TR::Node * callNode,
                new (self()->trHeapMemory()) TR::MemoryReference(methodMetaDataVirtualRegister, fej9->thisThreadGetCurrentExceptionOffset(), self()->cg()));
 
    TR::Instruction *gcPoint = (TR::Instruction *) generateS390CompareAndBranchInstruction(self()->cg(),
```

Comment:
While we are changing this line, can you remove unnecessary type casting of the return value of `generateS390CompareAndBranchInstruction`? It already returns `TR::Instruction*`.